scripts/check_frontend_coverage.py

```diff
@@ -326,7 +326,7 @@ def check_coverage_changes():
         else:
             if total_lines == covered_lines:
                 errors += ('\033[1m{}\033[0m file is fully covered but it\'s'
-                ' supposed to have 100% coverage. Please add the file name in'
+                ' supposed to be in the whitelist. Please add the file name in'
```

Comment:
... but it's not in the "100% coverage" whitelist. Please ...

(also, sorry for missing this the first time around!)

Reply:
Done.